ta/arch/arm32/user_ta_header.c

```diff
@@ -35,6 +35,15 @@
 #define TA_TRACE_LEVEL_DEFAULT CFG_TRACE_LEVEL
```

Comment:
Another alternative of writing the code is:

    #ifndef TA_TRACE_LEVEL_DEFAULT
    int trace_level = CFG_TRACE_LEVEL;
    #else
    int trace_level = TA_TRACE_LEVEL_DEFAULT;
    #endif

I prefer the original way.